### PR TITLE
Add time attribute to widget

### PR DESCRIPTION
**Pull request by LenaWil**
Haven’t fully tested this on all platforms yet, hence the draft, but this makes a clock widget appear on Android when you click a time field.

I couldn’t figure out where the widgets are normally edited, so I hope I’m settings this in the right place.

### CODE REVIEW

**Comment by LenaWil** [comment]
I would like a small review though, just scared to merge it currently.

**Comment by LenaWil** [comment]
Okay, testing this on iOS safari, macOS Safari, Chrome, Firefox, and Android Firefox and Chrome, it seems to work.

**Review by mhvis — APPROVED** [review]
Good idea.

You can also put it in the `Meta` class as per https://docs.djangoproject.com/en/4.1/topics/forms/modelforms/#overriding-the-default-fields, e.g.:

```python
class Meta:
    widgets = {'serve_time': TimeInput(input_type='time')}
```

Edit: @DutcherNL you can merge it.

**Comment by LenaWil** [comment]
> You can also put it in the Meta class as per https://docs.djangoproject.com/en/4.1/topics/forms/modelforms/#overriding-the-default-fields.

Doesn’t work for DateTimeInput though, at least not this syntax.

File: dining/forms.py [hunk 1]
```diff
@@ -136,8 +139,35 @@ class Meta:
...
+        # self.fields['serve_time'].widget.attrs['min'] = settings.KITCHEN_USE_START_TIME.strftime("%H:%M")
+        self.set_bounds('serve_time', 'min', settings.KITCHEN_USE_START_TIME.strftime("%H:%M"))
+        # self.fields['serve_time'].widget.attrs['max'] = settings.KITCHEN_USE_END_TIME.strftime("%H:%M")
+        self.set_bounds('serve_time', 'max', settings.KITCHEN_USE_END_TIME.strftime("%H:%M"))
+        # self.fields['sign_up_deadline'].widget.attrs['max'] = self.instance.date.strftime("%Y-%m-%dT23:59")
+        self.set_bounds('sign_up_deadline', 'max', self.instance.date.strftime("%Y-%m-%dT23:59"))
+        # self.fields['min_diners'].max_value = settings.MAX_SLOT_DINER_MINIMUM
+        self.set_bounds('min_diners', 'max', settings.MAX_SLOT_DINER_MINIMUM)
+        # self.fields['max_diners'].min_value = settings.MIN_SLOT_DINER_MAXIMUM
+        self.set_bounds('max_diners', 'min', settings.MIN_SLOT_DINER_MAXIMUM)
```
Comment (LenaWil) [inline dining/forms.py:159]: Opinion about this?

**Comment by LenaWil** [comment]
Although it’s okay to merge 740f8680765051f4229b95d0bf86150ca34630bd now, the extra validation stuff is not that important.

File: dining/forms.py (hunk 1 above)
Comment (mhvis) [reply dining/forms.py:159]: Yes that is nice in terms of UX.

**Review by mhvis — APPROVED** [review]

**Comment by LenaWil** [comment]
@DutcherNL Can you merge this?

**Pull request merged**